incenp/grainyhead/metrics.py

```diff
@@ -354,20 +354,20 @@ def _write_line(self, label, reports, output, with_total):
 
         if with_total:
             total = getattr(reports[0], property_name)
-            output.write(f"| {label:20} | {total: 8} |")
+            output.write(f"| {label} | {total} |")
 
             for report in reports[1:]:
                 value = getattr(report, property_name)
                 if total > 0:
                     percent = value / total * 100
                 else:
                     percent = 0.0
-                output.write(f" {value:8} | {percent: 8.2f} |")
+                output.write(f" {value} | {percent} |")
```

Comment:
The formatting specifier after `percent` here serves not only to fix the column width but also to fix the floating point precision, to avoid displaying needlessly over-precise (to the point of being meaningless) values such as `60.46511627906976`.